vuepress/docs/next/docs/reference/entando-cli.md

```diff
@@ -5,50 +5,52 @@ sidebarDepth: 2
 
 ## Overview
 
-The Entando Command Line Interface (CLI) provides a set of commands that accelerate the developer experience by assisting the developer with common tasks such as quickly installing a new copy of Entando, generating an Entando project via JHipster, deploying an Entando Bundle, etc.
+The Entando Command Line Interface (CLI) provides a set of commands that accelerate the developer experience by assisting with common tasks such as installing a new copy of Entando, generating an Entando project via JHipster, deploying an Entando bundle, etc.
 
 ## Installation
 
 ### Prerequisites
 
-The basic requirements for the CLI vary depending on the category of developer tasks. The Entando CLI is able to install secondary dependencies using the `ent check-env` command as described [here](#check-environment).
+The basic requirements for the CLI vary depending on the category of developer tasks. The CLI is able to install secondary dependencies using the `ent check-env` command as described [here](#check-environment).
 
 | Category | Prerequisite
 | :-: | :-:
 |Basic Development| git or [git for windows](https://gitforwindows.org)
 | | nvm or [nvm for windows](https://github.com/coreybutler/nvm-windows)
 | Install Entando in a local VM | [multipass](https://multipass.run/#install)
 | Build and publish Entando Bundles | docker and docker-compose
-| | a git repository for the bundle artifacts
+| | a Git repository for the bundle artifacts
 | | a Docker Hub account (or organization) for microservice Docker images
-| Deploy an Entando Bundle | a Kubernetes cluster with admin access. This could be a local cluster (created via the CLI or manually) or a shared remote cluster.
+| Deploy an Entando Bundle | a Kubernetes cluster with admin access, i.e. a local cluster (created via the Entando CLI or manually) or a shared remote cluster
```

Comment:
minor thing, I prefer the separate sentences because international people may have trouble with i.e. Not sure but just a thought.

Reply:
i hadn't thought of that! do you think e.g. might also not translate well? i wanted one sentence to avoid the use of a period / for consistency... i'll switch out i.e. for "such as"

Reply:
i think the types are limited to those two options so used "which can be"